### DISCUSSION TITLE
Discriptence between tab achievements and achievements on main section

### DISCUSSION THREAD

**Discussion by alex-semenyuk** [community/community#60125] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Discriptence between tab achievements and achievements on main section. The pull shark shows x2 and x3 accordance. Should be x3 https://github.com/alex-semenyuk?tab=achievements

**Comment by Vicwxy9** [comment]
https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it
